identity-apps-core/apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties

```diff
@@ -611,3 +611,7 @@ error.duo.user.not.found=Couldn't get the verified user name.
 error.mobile.not.found=User may not have a mobile number in user's profile for Duo Authentication. Please update it.
 error.mobile.not.found.duo=Unable to get the registered phone number from Duo. Authentication failed.
 error.number.mismatch.duo=Authentication failed due to mismatch in mobile numbers. Please update the correct registered duo mobile number in user profile.
+
+# Conditional authentication scripts
+prompt.template.internal.wait.greeting=Hello There!
```

Comment:
Are we providing support to customize this text through the adaptive script?

Reply:
Yes. But i18n support is there only for the default text

Reply:
Did we review these document content ?

Reply:
This is demonstrated in a review session. @himeshsiriwardana can you also review the content?

Reply:
I think we can go with

**Just a moment..**

**We are setting things up for you**

Reply:
Updated